x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -251,6 +251,20 @@ export class Simulator {
     return this.findInDOM('[data-test-subj="resolver:graph"]');
   }
 
+  /**
+   * The outer panel container.
+   */
+  public panelElement(): ReactWrapper {
+    return this.findInDOM('[data-test-subj="resolver:side-panel"]');
```

Comment:
I would be consistent about the naming here and below. Either `resolver:panel` or `resolver:side-panel`

Reply:
Good call, I could call the outer one `resolver:panel` and the inner ones like 'resolver:panel:process-list'. That would probably look better, right?